ha/k8s_setup/const.py

```diff
@@ -25,6 +25,9 @@
 #Confstore delimiter
 _DELIM=">"
 
+# consule endpoint scheme: http
+consule_scheme = 'http'
```

Comment:
Please check spelling. It should be "consul_scheme"

Reply:
done

Reply:
Resolved.